modules/usrmgr/mod_settings/translation.xml

```diff
@@ -14,22 +14,28 @@
 		<content>
 			<entry>
 				<id>ff16eac4bb1728bdfea6162a83b34d65</id>
-				<org>Dein Passwort und weitere Informationen wurden an deine angegebene E-Mail-Adresse gesendet.</org>
+				<org>Dein Passwort und weitere Informationen wurden an deine angegebene

```

Comment:
Are these word-wraps intentional or auto-formatting gone wrong?

Reply:
No, those are not intentional. Sorry, this seem to be a unwanted behaviour of my editor (VS Code). Others seem to have the same challenge: see https://stackoverflow.com/questions/6535831/how-to-stop-visual-studio-from-wrapping-lines-when-xml-is-auto-generated-for-ser

Do you consider this a blocker ?
Then I would try to revert this.
Otherwise, I would suggest we run a PR where we reformat all XML files towards a unified way (many of them have quirks).

You can review the PR without whitespace changes, see https://github.blog/2018-05-01-ignore-white-space-in-code-review/

Reply:
> Do you consider this a blocker ?

I wouldn't want to merge this if line-breaks are visible on output.
If they are not, then it does not matter

Reply:
Line-Breaks are not visible in output, thus fine to merge